vip_spec.xsd

```diff
@@ -259,7 +259,7 @@ reference elements for polling location
       <xs:element name="PostElectionStatus" type="CandidatePostElectionStatus" minOccurs="0" />
       <xs:element name="PreElectionStatus" type="CandidatePreElectionStatus" minOccurs="0" />
       <!-- NOTE: Still trying to get feedback or an example of what this means and how it's used -->
-      <xs:element name="SequenceOrder" type="xs:integer" />
+      <xs:element name="SequenceOrder" type="xs:integer" minOccurs="0" />
```

Comment:
Yeah, basically I'm doing the minimum amount of work just to get the schema to a point where a feasible sample feed can validate.